lib/freno/client/request.rb

```diff
@@ -37,7 +37,7 @@ def perform
       protected
 
       def request(verb, path)
```

Comment:
```suggestion
      def request(verb, path, params)
```

hopefully I'm understanding this properly.

Reply:
Interesting! It works without this but I'll make a couple of small changes to support the current style. Nice catch!

Reply:
👌 Pushed a1308b349f96d327e30cc9f5d757b8173c88f4ae!